src/main/scanCache/scanCacheManager.ts

```diff
@@ -1,16 +1,38 @@
-import { IArtifact } from 'jfrog-client-js';
+import * as fs from 'fs';
+import { IArtifact, IIssue, ILicense } from 'jfrog-client-js';
+import * as path from 'path';
 import * as vscode from 'vscode';
 import { ExtensionComponent } from '../extensionComponent';
+import { Issue } from '../types/issue';
+import { License } from '../types/license';
+import { INodeInfo } from '../types/nodeInfo';
+import { Translators } from '../utils/translators';
 import { ScanCacheObject } from './scanCacheObject';
 
 /**
  * Provide the scan results cache in a key-value style map.
```

Comment:
I suggest we extend the ```ScanCacheManager``` description and add the following informartion:
1. Why do we need this the cache manager? What challenge does it overcome?
2. How does the cache work? Which information does it store? Where is the information stored and what the format?
3. How can the cache be used? Let's add usage examples.